projects/ngrx.io/content/guide/eslint-plugin.md

```diff
@@ -0,0 +1,79 @@
+# ESLint plugin for NgRx
+
+## What is it?
+
+Use [ESLint](https://eslint.org/) to follow the best practices and to avoid common pitfalls in your application.
+
+The [NgRx ESLint Plugin](https://github.com/timdeschryver/eslint-plugin-ngrx) is no different and promotes the key concepts to create a maintainable project. It consists of @ngrx/store, @ngrx/effects, and @ngrx/component-store rules and a handful of preconfigured configurations.
+
+For more information, see the [documentation page](https://github.com/timdeschryver/eslint-plugin-ngrx/blob/main/README.md).
```

Comment:
This link is already added in the next line, so I think we could remove this one.

```suggestion
```

Reply:
Also a good point